Sources/Sentry/SentryCrashStackEntryMapper.m

```diff
@@ -31,18 +34,26 @@ - (SentryFrame *)sentryCrashStackEntryToSentryFrame:(SentryCrashStackEntry)stack
 
     frame.instructionAddress = sentry_formatHexAddressUInt64(stackEntry.address);
 
-    frame.imageAddress = sentry_formatHexAddressUInt64(stackEntry.imageAddress);
-
     if (stackEntry.symbolName != NULL) {
         frame.function = [NSString stringWithCString:stackEntry.symbolName
                                             encoding:NSUTF8StringEncoding];
     }
 
-    if (stackEntry.imageName != NULL) {
-        NSString *imageName = [NSString stringWithCString:stackEntry.imageName
-                                                 encoding:NSUTF8StringEncoding];
-        frame.package = imageName;
-        frame.inApp = @([self.inAppLogic isInApp:imageName]);
+    BinaryImageInfo *info = [_binaryImageCache imageByAddress:stackEntry.address];
```

Comment:
`h`: Can't we use `stackEntry.imageAddress` here? Then we could get the binaryImageInfo from a dictionary which would be `O(1)` instead of doing a binary search `O(log n)`. If not, please add a comment to explain why.

Reply:
`stackEntry.imageAddress` will be 0 if debug is disable, the only thing stackEntry has is `address`.
We also need the image name for the package property. 
But we could check if both informations are already available before looking into the cache.

Reply:
Please add a comment for that.